addons/odoo_payments/models/adyen_account.py

```diff
@@ -194,6 +216,67 @@ def _check_vat(self):
                     "The given registration number is invalid: %s",
                     account.registration_number))
 
+    def _compute_payment_journal_id(self):
+        for account in self:
+            acquirer = self.env['payment.acquirer'].search([
+                ('provider', '=', 'odoo'),
+                ('company_id', '=', account.company_id.id),
+            ], limit=1)
+            account.payment_journal_id = acquirer.journal_id
+
+    def _inverse_payment_journal_id(self):
+        for account in self:
+            acquirer = self.env['payment.acquirer'].search([
+                ('provider', '=', 'odoo'),
+                ('company_id', '=', account.company_id.id),
+            ], limit=1)
+            acquirer.journal_id = account.payment_journal_id
```

Comment:
What is this inverse for???
An inverse should modify another model, not reset the field.  You probably want a compute readonly=False, without an inverse